aten/src/ATen/core/ATenOpList.h

```diff
@@ -0,0 +1,14 @@
+#pragma once
+
+#include <c10/macros/Export.h>
+
+namespace c10 {
+struct OperatorName;
+}
+
+namespace at {
+
+// list of ATen ops that come from native_functions.yaml
+CAFFE2_API bool is_aten_op(const c10::OperatorName& opName);
```

Comment:
This new name doesn't seem very descriptive. (A better question to ask, perhaps: why do I care if it `is_aten_op`? Why isn't this the same thing as just checking the namespace?)

Reply:
Agreed, the implications of this name aren't that clear unless you're pretty deep into what's going on. Judging by its use [here](https://github.com/pytorch/pytorch/pull/36838/files#diff-94205457152f6821c30329a4ddb40bbeR172), I'd suggest maybe inverting the sense and making it `is_custom_op` instead.

Reply:
checking the namespace is a good idea, we should move towards doing that instead. I added a TODO. It likely doesn't work just yet because we might have internal "custom" ops that use the aten namespace.

Reply:
@smessmer when I get everything onto the new API, we will be able to guarantee that custom ops don't use the aten namespace.

But I am kind of curious what the downstream usage of this is, because the way I'm likely going to make the internal "Custom" ops (you're talking about the JIT ops right?) live in a centralized location is by hardcoding their registrations in TypeDefault.cpp ala https://github.com/pytorch/pytorch/issues/36925

See also https://github.com/pytorch/pytorch/issues/36926